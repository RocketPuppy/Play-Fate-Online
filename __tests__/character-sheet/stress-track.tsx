import * as React from "react";
import * as renderer from "react-test-renderer";
import StressTrack from "../../src/character-sheet/stress-track";

test("renders with the track name", () => {
  const render = renderer.create(<StressTrack trackName="stress track" />);

  expect(render.toJSON()).toMatchSnapshot();
});

