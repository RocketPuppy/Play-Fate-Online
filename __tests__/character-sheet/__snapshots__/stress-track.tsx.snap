// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with the track name 1`] = `
<div>
  <h2>
    stress track
  </h2>
  <div>
    <label>
      <span>
        1
      </span>
      <input
        checked={false}
        onChange={[Function]}
        type="checkbox"
      />
    </label>
    <label>
      <span>
        2
      </span>
      <input
        checked={false}
        onChange={[Function]}
        type="checkbox"
      />
    </label>
    <button
      onClick={[Function]}
    >
      Add Stress
    </button>
  </div>
</div>
`;
